### ISSUE TITLE
chat tool invocation input info is not present in the accessible view

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#249280]
We should have the input here 

<img width="940" alt="Image" src="https://github.com/user-attachments/assets/356ba573-ebd7-4775-b48f-e7062f275cd8" />

**Comment by anthonykim1** [comment]
this is great! 
Verified:

<img width="1116" alt="Image" src="https://github.com/user-attachments/assets/541e03c4-feb0-4091-ad8c-ef989c612397" />
